simulators/ethereum/engine/testenv.go

```diff
@@ -115,7 +115,7 @@ func RunTest(testName string, ttd *big.Int, slotsToSafe *big.Int, slotsToFinaliz
 	}()
 
 	// Setup timeouts
-	env.Timeout = time.After(timeout)
+	env.Timeout = time.After(2 * timeout)
```

Comment:
I think it would be better to increment the timeout only for the tests that require this double value.

Reply:
Also, this timeout was just recently updated in https://github.com/ethereum/hive/commit/7ef78bf4723fc1199d6b276b30b231394aaae4b9